lesson7..html

```diff
@@ -5,76 +5,181 @@
+        <div>
+            <label for=""> Age:</label>
+            <input type="number" placeholder="Nhap tuoi" id="age">
         </div>
-        <div class="input-box3">
-            <label for="Address 2">Address 2</label>
-            <br>
-            <input type="text" placeholder="1234 Viet Nam">
+        <div>
+            <button onclick="add()">Submit</button>
         </div>
-        <div class="input-box4">
-                <div class="col-6">
-                    <label for="city">City</label>
-                    <br>
-                    <input type="text">
-                </div>
-                <div class="col-6">
-                    <label for="State">State</label>
-                    <br>
-                    <input type="text">
-                </div>
+       <!-- <div>
+            <input type="text" placeholder="Nhap ten can tim" id="searchItem">
         </div>
-        <div class="clearfix"></div>
-        <div class="input-box5">
-            <input type="checkbox" id="Check me out">
-            <label for="Check me out">Check me out</label>
+        <div>
+            <button onclick="search()"> Search</button>
+        </div>-->
+        <table class="table" id="render">
+            <thead>
+                <tr>
+                    <th>Id</th>
+                    <th>Name</th>
+                    <th>Age</th>
+                </tr>
+            </thead>
+            <tbody id="body"></tbody>
+        </table>
+        <div class="dropdown">
+            <button class="btn btn-secondary dropdown-toggle text-white p-2 bg-secondary ">Order</button>
+            <div class="dropdown-content">
+                <a href="#" onclick="orderName()">Name</a>
+                <a href="#" onclick="orderAge()">Age</a>
+            </div>
         </div>
-        <button disabled id="button">
-          <a href="#" class="button">Sign in</a>
-        </buton>
-    </div>
-    <script>
-function checkValue() {
-let email = document.getElementById('email').value;
-let password = document.getElementById('password').value; 
-if (email != "" && password != "") {
-//show btn
-document.getElementById('button')./*style.display= 'block'*/disabled = fasle; //hien
-} else {
-    //hide btn
-document.getElementById('button')./*style.display='none'*/disabled = true; //an
-}
-}
+        <script>
+            var data = [
+                {
+                    Id: 112,
+                    Name: 'Nguyen Van B',
+                    Age: 18,
+                },
+                {
+                    Id: 234,
+                    Name: 'Nguyen Van D',
+                    Age: 14,
+                },
+                {
+                    Id: 245,
+                    Name: 'Nguyen Van A',
+                    Age: 53,
+                },
+                {
+                    Id: 345,
+                    Name: 'Nguyen Van C',
+                    Age: 34,
+                }
+            ]
+            function showInfodata(abc) {
```

Comment:
```suggestion

            function showInfoData(abc) {
```
Trước khi mở khối thì nên có 1 line trắng
hàm camelCase nhé